index.js

```diff
@@ -59,15 +59,25 @@
           reject('No year end found.')
         }
       }).catch(function(err) {
-          reject('Problem with reading file', err);
+          reject('Problem with reading file\n' + err);
       })
     })
 
     // Utility functions
     function loadField(conceptToFind, fieldName, key) {
       key = key || '$t';
       fieldName = fieldName || conceptToFind;
-      self.fields[fieldName] = _.get(self.documentJson, ['dei:' + conceptToFind, key], 'Field not found.');
+      var concept = _.get(self.documentJson, 'dei:' + conceptToFind);
+      //console.log(fieldName + "=> " + JSON.stringify(concept, null, 3));
+      if(_.isArray(concept)) {
+        // Default to the first available contextRef
+        console.warn('Found ' + concept.length + ' context references')
+        _.forEach(concept, function(c, idx) { console.warn('=> ' + c.contextRef + (idx === 0 ? ' (selected)' : ''))});
+        concept = _.find(concept, function(c, idx) { return idx === 0; });
+      }
+      self.fields[fieldName] = _.get(concept, key, 'Field not found.');
+      
+      console.log(`loaded ${fieldName}: ${self.fields[fieldName]}`);
```

Comment:
This syntax would break in node < v4.x or browsers not supporting ES6 (see [`template literals` here](https://kangax.github.io/compat-table/es6/)).   Does `parse-xbrl` currently support the browser?   We could add an [`engines`](https://docs.npmjs.com/files/package.json#engines) element to the `package.json` to express the node dependency or, if this syntax concerns you for backward compat reasons it's really just syntatic sugar that could be replaced with ES5 string concatenation.